ucsschool/kelvin/client/user.py

```diff
@@ -131,7 +130,14 @@ def __init__(
         url: str = None,
         session: Session = None,
     ):
-        super().__init__(name=name, ucsschool_roles=ucsschool_roles, dn=dn, url=url, session=session)
+        super().__init__(
+            name=name,
+            ucsschool_roles=ucsschool_roles,
+            dn=dn,
+            url=url,
+            session=session,
+            udm_properties=udm_properties,
```

Comment:
Keep order of argument consistent. `udm_properties` follows `ucsschool_roles` and is before `dn`.